packages/fx-core/tests/component/driver/file/generateAppsettings.test.ts

```diff
@@ -69,12 +68,13 @@ describe("AppsettingsGenerateDriver", () => {
       if (result.isErr()) {
         chai.assert(result.error instanceof InvalidParameterUserError);
         const message =
-          "Following parameter is missing or invalid for file/updateAppSettings action: appsettings.";
+          "Following parameter is missing or invalid for file/updateJson action: appsettings.";
```

Comment:
just to double confirm the last word is still appsettings?

Reply:
Yes, since the parameter schema does not change. This action is to generate some appsettings, in JSON format.
``` yaml
  - uses: file/updateJson
    with:
      target: some.json
      appsettings: # still appsettings
        A: ${{SOME_ENV}}
        B: 
          Foo: Bar
        C:
          - true
          - false
```